shared/languages/pythonPyodide.ts

```diff
@@ -109,9 +109,9 @@ function exceptionRewriter(msg: string, _: string) {
   });
 }
 
-export const PYTHON = {
-  name: 'Python',
-  defaultPath: 'python3',
+export const PYTHON_PYODIDE = {
+  name: 'Python (pyodide)',
```

Comment:
I'll need to think about this because this should only be shown in in-memory mode. When you're running the desktop app it does not use the inMemoryEval code and does not use wasm. It uses your computer's python3.